tests/io/test_parquet.py

```diff
@@ -201,7 +201,18 @@ def test_parquet_helper(data_and_type, use_daft_writer):
             before = before
             before = before.sort(col("_index"))
             before.write_parquet(file_path)
-            after = daft.read_parquet(file_path).sort(col("_index"))
+            after = daft.read_parquet(file_path)
+            after.limit(50).collect()
```

Comment:
unneeded here since we compare after with limit

Reply:
Good catch, removed